src/main/webapp/components/SecurityReview.js

```diff
@@ -0,0 +1,219 @@
+        id: "inputSensitive",
+        name: "textSensitive",
+        label: "Please explain*",
+        value: this.props.review === true ?  this.props.formData.projectExtraProps.textSensitive : this.props.currentValue.securityInfoFormData.textSensitive,
+        currentValue: this.props.review === true ?  this.props.current.projectExtraProps.textSensitive : undefined,
+        disabled: false,
+        required: false,
+        onChange: this.handleInputChange,
+        error: this.props.infoSecurityErrors.textSensitive && this.props.showErrorInfoSecurity,
+        errorMessage: "Required field",
+        readOnly: this.props.readOnly,
+        edit: this.props.edit,
+        review: this.props.review
+      }),
+      InputFieldRadio({
+        id: "radioAccessible",
+        name: "accessible",
+        label: span({}, ["Will the data collected or generated as part of this project be made available in an unrestricted/open-access environment ", span({ className: 'normal' }, ["(e.g. publicly available on the internet, shared via an open access repository such as GEO, etc)"]), "?*"]),
+        value: this.props.review === true ? this.props.formData.projectExtraProps.accessible : this.props.currentValue.securityInfoFormData.accessible,
+        currentValue: this.props.review === true ? this.props.current.projectExtraProps.accessible : null,
+        optionValues: ["true", "false", "uncertain"],
+        optionLabels: [
+          "Yes",
+          "No",
+          "Uncertain"
+        ],
+        onChange: this.handleRadio2Change,
+        required: true,
+        error: this.props.infoSecurityErrors.accessible && this.props.showErrorInfoSecurity,
+        errorMessage: "Required field",
+        readOnly: this.props.readOnly,
+        edit: this.props.edit,
+        review: this.props.review
+      }),
+      InputFieldText({
+        isRendered: this.props.review === true ?  this.props.formData.projectExtraProps.accessible === 'true' : this.props.currentValue.securityInfoFormData.accessible === 'true',
+        id: "inputAccessible",
+        name: "textAccessible",
+        label: "Please explain*",
+        value: this.props.review === true ? this.props.formData.projectExtraProps.textAccessible : this.props.currentValue.securityInfoFormData.textAccessible,
+        currentValue:  this.props.review === true ? this.props.current.projectExtraProps.textAccessible : undefined,
+        disabled: false,
+        required: false,
+        onChange: this.handleInputChange,
+        error: this.props.infoSecurityErrors.textAccessible && this.props.showErrorInfoSecurity,
+        errorMessage: "Required field",
+        readOnly: this.props.readOnly,
+        edit: this.props.edit,
+        review: this.props.review
+      })
+    ])
+  };
+
+  render() {
+    if (this.state.hasError) {
+      // You can render any custom fallback UI
+      return h1({}, ["Something went wrong."]);
```

Comment:
This seems pretty unhelpful. Can we add something explanatory here?

Reply:
I agree. This should be more explanatory. But we're using this as a simple way to display a error when the component can't load. Probably we need to fix this later to a better explanation, in all the components this messages is present. At least for us this message it's helpful for now.